saleor/core/tasks.py

```diff
@@ -1,8 +1,20 @@
 from django.core.files.storage import default_storage
+from django.utils import timezone
 
 from ..celeryconf import app
+from ..settings import EVENT_PAYLOAD_DELETE_PERIOD
+from .models import EventDelivery, EventDeliveryAttempt, EventPayload
 
 
 @app.task
 def delete_from_storage_task(path):
     default_storage.delete(path)
+
+
```

Comment:
Should we add `@app.task` here?

Reply:
Yes this was a task before and it should be a task.